documentacion/onepay/README.md

```diff
@@ -226,7 +226,36 @@ cart.add(Item(description="Zapatos",
               additional_data=None, expire=None))
 ```
 
-El monto en el carro de compras debe ser positivo, en caso contrario se lanzará una excepción.
+Se permite agregar un `Item` al carro con amount negativo que generalmente representa un descuento. Lo importante es que el monto total en el carro de compras debe ser positivo, en caso contrario se lanzará una excepción.
+
+```php
```

Comment:
The snippets should go *before* the text they are linked with. See https://github.com/lord/slate/wiki/Markdown-Syntax#formatting-code-annotations-right-column-with-its-explanation-center-column